bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -3079,13 +3078,13 @@ app_action_button_down                = &Down
 app_action_button_up                  = &Up
 app_action_collapse_all_tooltip       = &Collapse All
 app_action_collapse_others_tooltip    = Collapse &Others
-app_action_delete_tour_marker         = Delete &marker(s)...
+app_action_delete_tour_marker         = Delete &Markers...
 app_action_edit_adjust_altitude       = Ad&just Altitude...

```

Comment:
Existing english texts should not be changed because translators do not see these changes